src/platform/library/schedule/ll_schedule.c

```diff
@@ -28,140 +27,42 @@ DECLARE_SOF_UUID("ll-schedule", ll_sched_uuid, 0x77de2074, 0x828c, 0x4044,
 
 DECLARE_TR_CTX(ll_tr, SOF_UUID(ll_sched_uuid), LOG_LEVEL_INFO);
 
-struct ll_vcore {
-	struct list_item list; /* list of tasks in priority queue */
-	pthread_mutex_t list_mutex;
-	pthread_t thread_id;
-	int vcore_ready;
-	int core_id;
-};
-
-static int tick_period_us;
+/* list of all tasks */
+static struct list_item sched_list;
 
-/**
- * Implement an override of how cores defined in SOF topology
- * are mapped to host cores.
- *
- * Returns the host core number to which SOF DSP core0 should
- * be mapped to.
- */
-static unsigned int sof_host_core_base(void)
+void schedule_ll_run_tasks(void)
```

Comment:
ok, from here, there is no relation with zephyr, it is an static functional test, use old path, with ipc3, right?

Reply:
Yes, this is a simplified scheduler for library platform. I hope we can get IPC4 support. It's a major work topic for testbench.